common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -109,6 +135,12 @@ def get_transcripts_from_youtube(youtube_id, settings, i18n):
 
     youtube_text_api = copy.deepcopy(settings.YOUTUBE['TEXT_API'])
     youtube_text_api['params']['v'] = youtube_id
+    # if the transcript name is not empty on youtube server we have to pass
+    # name param in url in order to get transcript
+    # example http://video.google.com/timedtext?lang=en&v={VideoId}&name={transcript_name}
+    youtube_transcript_name = transcript_name_youtube_video(youtube_text_api)
```

Comment:
Should we check to see if `youtube_text_api['params']['name']` is set before making a request to youtube for it? Or will it never be set at this point?